### PR TITLE
Add Google analytics integration, Privacy Policy and Licenses pages

### PR DESCRIPTION
**Pull request by jurgelenas**

### CODE REVIEW

File: docs/info/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+## 3. Information Collection and Use
+
+We collect several different types of information for various purposes to provide and improve our Service to you.
+
+## 4. Types of Data Collected
+
+While using our Service, we may ask you to provide us with certain personally identifiable information that can be used
+to contact or identify you (“Personal Data”). Personally identifiable information may include, but is not limited to:
+
+0.1. Email address
```
Comment (deadbytefpv) [inline docs/info/privacy-policy.md:54]: I don't think these sections apply on our website.

We don't collect these in any shape or form. 
we can leave just the Usage Data, but I don't think personal info are collected.

File: docs/info/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+
+Examples of Cookies we use:
+
+0.1. **Session Cookies**: We use Session Cookies to operate our Service.
+
+0.2. **Preference Cookies**: We use Preference Cookies to remember your preferences and various settings.
+
+0.3. **Security Cookies**: We use Security Cookies for security purposes.
+
+### Other Data
```
Comment (deadbytefpv) [inline docs/info/privacy-policy.md:100]: Are we collecting these?

**Review by deadbytefpv — COMMENTED** [review]
If we only need to comply, and the Generic Privacy Policy is good enough for you guys, then disregard my comments.

I just think that we should cater this document for our specific use-case and target audience.

File: docs/info/privacy-policy.md (hunk 2 above)
Comment (jurgelenas) [reply docs/info/privacy-policy.md:98]: Github Pages might use cookies when defending against bots. Same applies to us if we device to use Cloudflare service in front of the ExpressLRS documentation.

File: docs/info/privacy-policy.md (hunk 1 above)
Comment (jurgelenas) [reply docs/info/privacy-policy.md:52]: This is just future proofing. For example, if we decide to add newsletter subscription form this section would be useful.

**Comment by jurgelenas** [comment]
> If we only need to comply, and the Generic Privacy Policy is good enough for you guys, then disregard my comments.
> 
> I just think that we should cater this document for our specific use-case and target audience.

I think this generic Privacy policy is good enough for our use case. It is not too broad and allows us to add more features to the ExpressLRS.org without changing the Privacy Policy itself.

**Review by deadbytefpv — APPROVED** [review]

**Review by maybenikhil — APPROVED** [review]

**Pull request merged**